test/TestProjects/OldestVersion-TypeSpec/src/Generated/Models/ExportedResource.cs

```diff
@@ -85,5 +90,7 @@ internal ExportedResource()
         public string ResourceUri { get; }
         /// <summary> Gets the name. </summary>
         public string Name { get; }
+        /// <summary> Gets the type. </summary>
+        public string Type { get; }
```

Comment:
@typeSpec/Versioning resolve issue https://github.com/microsoft/typespec/issues/3239, so the `Type` Field should be generated.